### DISCUSSION TITLE
What if no alertmanager servers specified in CONFIG_FILE?

### DISCUSSION THREAD

**Discussion by sed-i** [prymitive/karma#3362] [General]
Currently karma exits if `alertmanagers.servers` section is blank. Does it make sense instead to start karma regardless? And maybe list somewhere in the UI the alertmanager servers connected to?

**Comment by prymitive** [comment]
How would you use karma instance with no alertmanagers configured?

**Reply by sed-i** [reply]
Here's the scenario:

1. the contents of CONFIG_FILE is not fully known when karma starts
2. karma is launched separately from (multiple clusters of) alertmanager
3. alertmanager cluster(s) can come and go

If karma would load even without alertmanagers configured, and would allow hot-reload (prometheus and alertmanager for example have `/-/reload` endpoint) then the scenario above would be somewhat easier to handle.

**Reply by primeroz** [reply]
Is there a reload endpoint on karma or any other way to reload the configuration ?
